autoupdate.nix

```diff
@@ -13,7 +13,7 @@
       git clone --depth=1 https://github.com/FabLab-Altmuehlfranken/NixOS-Workstation.git /etc/nixos
       nixos-rebuild boot --flake '/etc/nixos#fablab'
       touch /autoupdate-finished
-      if [ $(TZ=UTC date +%H) = 3 ]; then shutdown -hP now; fi
+      if [ $(TZ=UTC date +%H) = 03 ]; then shutdown -hP now; fi
```

Comment:
I verified this with

```
$ TZ='Australia/Sydney' date +%H
03
```

Not sure if this is more intuitive, but would not require the leading zero:
```
$ TZ='Australia/Sydney' date '+%-H'
3
```